.github/workflows/test.yml

```diff
@@ -4,6 +4,14 @@ on:
   pull_request:
     types: [opened, reopened, edited, synchronize]
 
+permissions:
+  actions: write
```

Comment:
Would you let me know why we need these permission

Reply:
I added these permissions to resolve this issue from Danger-Kotlin
```
Resource Not Available to Integration
```
Following the [solution](https://github.com/danger/danger-js/issues/1031#issuecomment-1017832635)